cli-plugins/apps/tutorials/get-workload-status.hbs.md

```diff
@@ -0,0 +1,635 @@
+   tanzu-java-web-app-config-writer-7xr6m-pod  0/1     Completed   0          60d
+   tanzu-java-web-app-config-writer-g9gp8-pod  0/1     Completed   0          45d
+
+🚢 Knative Services
+   NAME                READY   URL
+   tanzu-java-web-app  Ready   http://tanzu-java-web-app.default.127.0.0.1.nip.io
+
+To see logs: "tanzu apps workload tail tanzu-java-web-app --namespace development --timestamp --since 1h"
+```
+
+## Tail logs
+
+### tanzu apps workload tail
+
+`tanzu apps workload tail` checks the runtime logs of a workload. The workload can be filtered
+by the namespace it was created on by using the `--namespace` flag, and the workload logs
+can also be filtered by a specific component. 
+
+To retrieve logs within a specific timeframe and display them with their timestamps, 
+you can utilize the --since flag.
+
+```bash
+tanzu apps workload tail tanzu-java-web-app -n development --since 1h --component build --timestamp
+
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.236584161-05:00 Build reason(s): CONFIG
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237805020-05:00 CONFIG:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237847016-05:00       + env:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237853995-05:00       + - name: BP_OCI_SOURCE
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237857982-05:00       +   value: tap-1.6.0/bb9e655b33e39b242b6e5d9e218578e75c1d833c
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237861706-05:00       resources: {}
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237864954-05:00       - source: {}
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237868502-05:00       + source:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237871941-05:00       +   blob:
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237875932-05:00       +     url: http://fluxcd-source-controller.flux-system.svc.cluster.local./gitrepository/default/tanzu-java-web-app/bb9e655b33e39b242b6e5d9e218578e75c1d833c.tar.gz
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237880878-05:00       +   subPath: tanzu-java-web-app
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.237884317-05:00 Loading registry credentials from service account secrets
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.238417749-05:00 Loading secret for "gcr.io" from secret "registry-credentials" at location "/var/build-secrets/registry-credentials"
+tanzu-java-web-app-build-1-build-pod[prepare] 2023-06-15T10:45:13.238437378-05:00 Loading secret for "https://gcr.io/" from secret "registry-credentials" at location "/var/build-secrets/registry-credentials"
+...
+```
+
+### tail a workload while creating/applying
+
+You can enable the ability to tail a workload during its creation or update by using the `--tail` flag
```

Comment:
You can tail a workload directly as part of the `tanzu apps workload create` or `tanzu apps workload apply` commands by providing the `--tail` flag and you can display timestamps for each log message by including the `--tail-timestamp` flag.

Reply:
changed